### PR TITLE
ml/add database table

### PR DESCRIPTION
**Pull request by alextran1502**
- Add class entities
- Generate migration file
- Update test fixture

image

### CODE REVIEW

File: server/libs/infra/src/entities/asset-face.entity.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export class AssetFaceEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string;
+
+  @Column({
+    type: 'float4',
+    array: true,
+    nullable: true,
+  })
+  embedded!: number[] | null;
```
Comment (jrasm91) [inline server/libs/infra/src/entities/asset-face.entity.ts:15]: should we call this "embedding", like we do for clip?

File: server/libs/infra/src/entities/person.entity.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  @Column()
+  thumbnailPath!: string;
+
+  @Column({
+    type: 'float4',
+    array: true,
+    nullable: true,
+  })
+  embedded!: number[] | null;
```
Comment (jrasm91) [inline server/libs/infra/src/entities/person.entity.ts:32]: I don't think we need this stored on person, do we?

**Review by jrasm91 — COMMENTED** [review]
Looking good!

File: server/libs/infra/src/entities/person.entity.ts (hunk 2 above)
Comment (alextran1502) [reply server/libs/infra/src/entities/person.entity.ts:32]: Do we need it to compare faces when an asset is uploaded?

Comment (jrasm91) [reply server/libs/infra/src/entities/person.entity.ts]: We need to store the embeddings, yes, but on the faces table. We could store a single, "central", embedding per person to use in calculations, but that's not the clustering approach we're going for. Instead we're going to try using typesense to find nearby faces and then use that data to assign the new face to a person.

File: server/libs/infra/src/entities/asset-face.entity.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { AssetEntity } from './asset.entity';
+import { PersonEntity } from './person.entity';
+
+@Entity('asset_faces')
+export class AssetFaceEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string;
```
Comment (jrasm91) [inline server/libs/infra/src/entities/asset-face.entity.ts:8]: This actually isn't needed. Asset id and person id should be the primary columns for this table.

**Pull request merged**